### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by Serabe**

### CODE REVIEW

File: tests/helpers/create-test-function.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { test } from 'qunit';
+
+export default function(fn) {
+    return function(given, expected, description) {
+        test(description, function (assert) {
+            assert.deepEqual(fn(given), expected);
+        })
+    }
+}
```
Comment (locks) [inline tests/helpers/create-test-function.js]: trailing newline

Comment (Serabe) [reply tests/helpers/create-test-function.js]: https://github.com/emberjs/ember-string/blob/master/.editorconfig#L12 ??

Comment (Serabe) [reply tests/helpers/create-test-function.js]: Now I see is not being consistent. WTHeck? Fixing it.

**Comment by Serabe** [comment]
Fixed.

**Pull request merged**